projects/core/src/process/store/selectors/process.selectors.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { TestBed } from '@angular/core/testing';
+import { select, Store, StoreModule } from '@ngrx/store';
+import * as fromActions from '../../../state/utils/entity-loader/entity-loader.action';
+import { LoaderState } from '../../../state/utils/loader/loader-state';
+import { PROCESS_FEATURE, StateWithProcess } from '../process-state';
+import * as fromReducers from '../reducers/index';
+import * as fromSelectors from './process.selectors';
+
+const MOCK_UPDATE_STATE = 'update-mock';
+
+describe('Cms Component Selectors', () => {
+  let store: Store<StateWithProcess<void>>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        StoreModule.forRoot({}),
+        StoreModule.forFeature(PROCESS_FEATURE, fromReducers.getReducers()),
+      ],
+    });
+    store = TestBed.get(Store);
+  });
+
+  describe('getProcessStateFactory', () => {
+    it('should return requested process slice of the state', () => {
+      store.dispatch(
+        new fromActions.EntitySuccessAction(PROCESS_FEATURE, MOCK_UPDATE_STATE)
+      );
+
+      let result: LoaderState<void>;
+      store
+        .pipe(select(fromSelectors.getProcessStateFactory(MOCK_UPDATE_STATE)))
+        .subscribe(value => (result = value))
+        .unsubscribe();
+
+      expect(result).toEqual({
+        loading: false,
+        success: true,
+        error: false,
+        value: undefined,
+      });
+    });
+  });
+
+  describe('getProcessLoadingFactory', () => {
+    it('should return loading flag', () => {
+      store.dispatch(
+        new fromActions.EntityLoadAction(PROCESS_FEATURE, MOCK_UPDATE_STATE)
+      );
+
+      let result: boolean;
```

Comment:
if we are going to assert that the boolean is true, perhaps set it explicitly to the opposite value before there selector call like `let result: boolean = false;`